.devcontainer/docker-compose.yml

```diff
@@ -8,15 +8,37 @@ services:
     volumes:
       - ../:/workspace/host-metering:Z
 
+  prometheus-rules-generator:
```

Comment:
I was getting `sh: can't create /workdir/recording-rules.yml: Permission denied`, I think the root cause was that the default user of `mikefarah/yq` is the `yq` user but `workdir` is writable only for root. 

When I added: `user: root` it started to work for me.